Nimble.xcodeproj/project.pbxproj

```diff
@@ -1601,6 +1603,7 @@
 				PRODUCT_NAME = Nimble;
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 2.3;
```

Comment:
This setting is already set at the project level, so should not be set at each target level.